varya/assets/sass/base/_editor.scss

```diff
@@ -22,11 +24,23 @@ a {
 	color: var(--global--color-primary);
 
 	&:hover {
-		color: var(--global--color-primary-hover);;
+		color: var(--global--color-primary-hover);
 	}
 }
 
 button,
 a {
 	cursor: pointer;
 }
+
+// Gutenberg injects a rule that limits the max width of .wp-block to 580px
+// This line overrides it to use the responsive spacing rules for default width content 
+.wp-block {
+	&:not([data-align="full"]):not([data-align="wide"]){
+		@extend %responsive-aligndefault-width;
```

Comment:
The left/right margins are causing nested blocks conflicts. 

If you replace the `%extend` with just the width setting, it’ll produce the right widths without the conflicting margin rules:
```
max-width: var(--responsive--aligndefault-width);
```

Reply:
Ahh gotcha. Do you think this is a reasonable place to put these rules for now?